collectors/0/secor_offset_collector.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from collectors.lib.kafka_offset_collector import KafkaOffsetCollector
+
+
+class SecorOffsetCollector(KafkaOffsetCollector):
```

Comment:
I left it in there for backwards compatibility, does secor still run on the Shadowfax Kafka? If not, I'm happy to delete this.